pkg/apis/core/v1beta1/types_seed.go

```diff
@@ -270,8 +270,23 @@ type SeedSettings struct {
 
 // SeedSettingExcessCapacityReservation controls the excess capacity reservation for shoot control planes in the seed.
 type SeedSettingExcessCapacityReservation struct {
-	// Enabled controls whether the excess capacity reservation should be enabled.
+	// Enabled controls whether the default excess capacity reservation should be enabled.
+	// Deprecated: This field is deprecated and will be removed in a future version of Gardener. Please use `Configs` instead.
```

Comment:
I'm wondering if we should really deprecate this field. We can make it a pointer and optional instead.

Reply:
In my opinion when we introduce the `Configs` structure `Enabled` is just another way to configure the same thing. Thus, I vote for deprecation.

Reply:
I agree if there is a convenient way to turn the feature off. Posting an empty config (rather than setting it to `nil` which is not possible because of defaulting) feels unnatural to me.

Reply:
How about keeping the `Enabled` flag to switch the feature on and off, but only create deployments for the configurations added to `Configs`?
When someone sets `Enabled` to `true` and `Configs` is empty, a default configuration will be created.

Reply:
Sounds good 👍 In addition, I'd make `Enabled` optional in order to have only `Configs` which implicitly means that the feature is enabled. WDYT?

Reply:
okay, let's go for it 😄